Update networking.md (#124843)
Azure/azure-content: articles/container-apps/networking.md

    | Subnet Size | Available IP Addresses<sup>1</sup> | Max nodes (Dedicated workload profile)<sup>2</sup>| Max replicas (Consumption workload profile)<sup>2</sup> |
    |--|--|--|--|
    | /23 | 495 | 247 | 2,470 |
    | /24 | 239 | 119 | 1,190 |
    | /25 | 111 | 55 | 550 |
    | /26 | 47 | 23 | 230 |
    | /27 | 15 | 7 | 70 |
    

>     <sup>1</sup> The available IP addresses is the size of the subnet minus the 12 IP addresses required for Azure Container Apps infrastructure minux the 5 IP addresses required per subnets.

Reviewer: ```suggestion
    <sup>1</sup> The available IP addresses is the size of the subnet minus the 12 IP addresses required for Azure Container Apps infrastructure and 5 IP addresses reserved by the subnet.
```
